src/main/scala/com.snowplowanalytics.snowplow.micro/model.scala

```diff
@@ -57,8 +57,15 @@ final case class TimelinePoint(validEvents: Int, failedEvents: Int, timestamp: I
 final case class TimelineData(points: List[TimelinePoint])
 
 /** Column statistics data structures for /micro/columnStats endpoint. */
-final case class ColumnStats(values: List[String])
+final case class ColumnStats(
+  sortable: Boolean,
+  filterable: Boolean,
+  values: Option[List[String]] = None
+)
 final case class ColumnStatsRequest(columns: List[String])
+final case class ColumnStatsResponse(
+  stats: Map[String, ColumnStats]
```

Comment:
Todo: undo this (keep the endpoint returning an object with column names as keys)